test/cbmc/sources/core_sntp_cbmc_state.c

```diff
@@ -90,16 +86,19 @@ SntpContext_t * allocateCoreSntpContext()
     {
         pAuthIntf->pAuthContext = malloc( sizeof( SntpAuthContext_t ) );
         pAuthIntf->generateClientAuth = GenerateClientAuthStub;
+        pAuthIntf->validateServerAuth = ValidateServerAuthStub;
     }
 
     /* It is part of the API contract to call Sntp_Init() with the SntpContext_t
      * before any other function in core_sntp_client.h. */
     if( pContext != NULL )
     {
+        pContext->numOfServers = numOfServers;
         pContext->currentServerIndex = currentServerIndex;
         sntpStatus = Sntp_Init( pContext, pTimeServers, numOfServers, serverResponseTimeoutMs, pNetworkBuffer,
                                 bufferSize, ResolveDnsFuncStub, GetTimeFuncStub, SetTimeFuncStub,
                                 pNetworkIntf, pAuthIntf );
+        pContext->sntpPacketSize = 4;
```

Comment:
Why is this set to 4? 

It would be fine to set it to any value that CBMC selects for it.

Reply:
This is set in Sntp_Init function to SNTP_PACKET_BASE_SIZE, which makes proof to run very slow, so stubbed it here.